lib/tests/bottles_fetch.rb

```diff
@@ -31,7 +31,9 @@ def fetch_bottles!(formula_name, args:)
 
         tags.each do |tag|
           cleanup_during!(args:)
-          test "brew", "fetch", "--retry", "--formulae", "--bottle-tag=#{tag}", formula_name
+          test "brew", "fetch", "--retry", "--formulae", "--bottle-tag=#{tag}",
+               "--concurrency", Homebrew::EnvConfig.make_jobs,
```

Comment:
What is `Homebrew::EnvConfig.make_jobs` in CI?

Reply:
Same as default -- number of cores. So somewhere between 2-6.